{"cveId": "CVE-2025-1834", "cwe": ["CWE-434", "CWE-284", "CWE-284", "CWE-434", "CWE-284", "CWE-434"], "cvss": [{"baseScore": "6.3", "version": "3.1", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "Low", "User Interaction": "None", "Scope": "Unchaged", "Confidentiality Impact": "Low", "Integrity Impact": "Low", "Availability Impact": "Low"}], "references": ["https://vuldb.com/?id.298101", "https://vuldb.com/?ctiid.298101", "https://vuldb.com/?submit.504842", "https://github.com/caigo8/CVE-md/blob/main/zz/zz_resolve%E6%8E%A5%E5%8F%A3%E4%BB%BB%E6%84%8F%E6%96%87%E4%BB%B6%E4%B8%8A%E4%BC%A0.md"], "description": ["A vulnerability, which was classified as critical, was found in zj1983 zz up to 2024-8. This affects an unknown part of the file /resolve. The manipulation of the argument file leads to unrestricted upload. It is possible to initiate the attack remotely. The exploit has been disclosed to the public and may be used. The vendor was contacted early about this disclosure but did not respond in any way.", "Es wurde eine kritische Schwachstelle in zj1983 zz bis 2024-8 gefunden. Hiervon betroffen ist ein unbekannter Codeblock der Datei /resolve. Dank Manipulation des Arguments file mit unbekannten Daten kann eine unrestricted upload-Schwachstelle ausgenutzt werden. Der Angriff kann über das Netzwerk angegangen werden. Der Exploit steht zur öffentlichen Verfügung."], "published": "2025-03-02T22:15:34.997", "state": "PUBLIC", "vendorName": ["zj1983"], "productName": ["zz"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}